spec/board_spec.rb

```diff
@@ -0,0 +1,55 @@
+require './lib/board'
+RSpec.describe Board  do
+
```

Comment:
First of all, use [let to share variables with different test cases](http://www.betterspecs.org/#let)